controller/scheduler/main.go

```diff
@@ -169,6 +181,7 @@ func (c *context) syncCluster() {
 				ReleaseID: releaseID,
 				Type:      jobType,
 				State:     "up",
+				Meta:      jobMetaFromMetadata(job.Metadata),
```

Comment:
I don't think we know whether the job is in the database yet, so we need to include the full details.